Ultisnips/arduino.snippets

```diff
@@ -21,7 +21,9 @@ extends cpp
 snippet setup "setup()"
 	void setup()
 	{
-		${1}
+                ${1}//Serial.begin(115200);
```

Comment:
I commented it out by default because you don't always need serial communication, but I also set the first point there so people can uncomment it very easily